modules/graphics/mrblib/font.rb

```diff
@@ -0,0 +1,17 @@
+module Moon
+  class Font
+    class << self
+      attr_accessor :default_shader
+    end
+
+    alias :initialize_wo_shader :initialize
+    def initialize(*args, &block)
+      initialize_wo_shader(*args, &block)
+      post_initialize
+    end
+
+    def post_initialize
+      self.shader = self.class.default_shader
```

Comment:
The C side need to set the underlying structure's shader, which is why its initialized afterwards, though, I could try pulling it from the shader method using `mrb_funcall()`, nice idea!